commons/src/main/java/org/codice/ddf/admin/commons/requests/RequestUtils.java

```diff
@@ -91,24 +76,26 @@
      * WARNING TYPES - UNTRUSTED_CA
      * FAILURE TYPES - CANNOT_CONNECT, CERT_ERROR
      * RETURN TYPES -  CONTENT_TYPE, CONTENT, STATUS_CODE
-     *
      * @param url URL to send Get request to
      * @param userName - Optional username to add to Basic Auth header
      * @param password - Optional password to add to Basic Auth header
      * @return report
      */
-    public ProbeReport sendGetRequest(String url, String userName, String password) {
-        Report connectReport = endpointIsReachable(url);
-        if (connectReport.containsFailureMessages()) {
-            return new ProbeReport(connectReport.messages());
+    public ProbeReport sendGetRequest(String url, String userName, String password, Map<String, String> queryParams) {
+
+        WebClient client = generateClient(url, userName, password);
+
+        for (Map.Entry<String, String> e : queryParams.entrySet()) {
+            client.query(e.getKey(), e.getValue());
         }
-        HttpGet request = new HttpGet(url);
+        Response r = client.get();
 
-        if (url.startsWith(HTTPS) && userName != null && password != null) {
-            String auth = Base64.getEncoder().encodeToString((userName + ":" + password).getBytes());
-            request.setHeader(HttpHeaders.AUTHORIZATION, "Basic " + auth);
+        if (r.getStatus() != HTTP_OK || r.readEntity(String.class).equals("")) {
+            LOGGER.warn("Bad or empty response.");
```

Comment:
Should this be warn? Seems more verbose than needed. Can we also add a better log msg? Something a long the lines of "Bad or empty response when performing get request to url: <url>"